src/lib/Linearize.hs

```diff
@@ -274,6 +274,7 @@ linearizeAtom atom = case atom of
   DepPairTy _     -> emitZeroT
   TC _            -> emitZeroT
   Eff _           -> emitZeroT
+  EffOp _ _ _     -> emitZeroT  -- TODO(alex): check correctness
```

Comment:
This feels sketchy... Are you sure there's no way to have a value we're differentiating in the free vars of an `EffOp`?

Reply:
Also, I guess this should just be unhandled. AD runs on simplified code, and simplified code should not contain any effectful operations.

Reply:
EffOp removed in #1070